modules/contact/doc/content/source/constraints/ComputeFrictionalForceCartesianLMMechanicalContact.md

```diff
@@ -0,0 +1,45 @@
+# ComputeFrictionalForceCartesianLMMechanicalContact
+
+!syntax description /Constraints/ComputeFrictionalForceCartesianLMMechanicalContact
+
+
+This class represents a preliminary implementation of frictional mortar contact constraints intended to be used with Lagrange's multiplier interpolation with dual bases. The nonlinear complementarity constraints employed here are based on a primal-dual active set strategy (PDASS), see [!citep](gitterle2010finite). These constraints capture nodes in sticking and slipping states on different solution branches, and can be written as:
+
+\begin{equation}
+C_{tj}(\lambda_{j},\boldsymbol{u}, \boldsymbol{\dot{u}}) = \max({\mu({pressure} + c_{n}\tilde{g}_{nj}),  abs({\lambda_{j} + c_t \tilde{u}_{tj}}))  \lambda_{j} - \mu \max({0,({pressure} + c_{n}\tilde{g}_{nj})}) (\lambda_{j} + c_t \tilde{u}_{tj})
+}
+\end{equation}
+
+$\lambda_{j}$ is a Lagrange's multiplier that refers to the tangential contact pressure at node $j$, $\tilde{u}_{tj}$ is the weighted tangential velocity integrated forward in time, $\tilde{g}_n)_j$ is the weighted normal gap, $c_{n}$ is a numerical parameter ($c$ in [`ComputeWeightedGapLMMechanicalContact`](/ComputeWeightedGapLMMechanicalContact.md)) and $c_{t}$ is a numerical parameter that can determine convergence properties but has no effect on the results.
```

Comment:
I think that we should not do `` formatting for class names that we want as links because it gets set as code and doesn't appear to the viewer as linkable

Reply:
Hmm It appears as linkable to me, at least, locally. Made that change throughout the mortar mechanical contact object documentation pages.

Reply:
Oh it is linkable for sure. It just doesn't appear as the standard blue hyperlink when it's set as code